env.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+envvars=( 'APP_URL' 'HMDA_API' 'KEYCLOAK_URL' )
+defaults=( 'http://192.168.99.100' 'https://192.168.99.100:4443/hmda' 'https://192.168.99.100:8443/auth/realms/hmda' )
```

Comment:
The reason the defaults are included is so devs working from the volumized compose setup don't have to set these envvars on their machines.